wrapping/itkLabelSetMorphBaseImageFilter.wrap

```diff
@@ -1,3 +1,7 @@
 itk_wrap_class("itk::LabelSetMorphBaseImageFilter" POINTER)
-  itk_wrap_image_filter("${WRAP_ITK_SCALAR}" 2 2+)
+  foreach(d ${ITK_WRAP_IMAGE_DIMS})
+    foreach(t ${WRAP_ITK_SCALAR})
+      itk_wrap_template("${ITKM_I${t}${d}}}${ITKM_B}" "${ITKT_I${t}${d}}}${ITKT_B}")
```

Comment:
As a reference: [itk_wrap_template](https://github.com/phcerdan/ITK/blob/master/Wrapping/TypedefMacros.cmake#L524-L530)

In the type side I think you are missing here the second image template: <TInputImage, bool, TOutputImage>
In the mangling name, I am not sure if it would be necessary because we are not allowing combinations of input and output image (we assume TOutputImage = TInputImage)

Also I think one extra `}` than needed? 
Missing commas in the right side for the different templates:
```cmake
       itk_wrap_template("${ITKM_I${t}${d}}${ITKM_B}"  "${ITKT_I${t}${d}}, ${ITKT_B}, ${ITKT_I${t}${d}}") 
```

Example in QuadEdgeMesh: https://github.com/phcerdan/ITK/blob/31ba947a5c235220744953900926bb5ba83775f9/Modules/Core/QuadEdgeMesh/wrapping/itkQuadEdgeMeshBase.wrap#L44-L49

It would require testing!

Reply:
Thanks @phcerdan ! Yes, just realized when I saw the CI result. Let's see what CI says about commit de81b4b.

Reply:
Nice, great to see the CI is on :+1:

Reply:
Well, now I ran out of ideas. 20d7f0e is still failing because the `bool` template argument is not being somehow correctly wrapped:
```
/work/ITK-cp27-cp27m-manylinux1_x64/Wrapping/itkLabelSetMorphBaseImageFilter.cxx:26:77: 
error: template argument for non-type template parameter must be an expression
    typedef itk::LabelSetMorphBaseImageFilter< itk::Image< signed short,2 >,bool,
itk::Image< signed short,2 > > itkLabelSetMorphBaseImageFilterISS2B;
                                                                            ^~~~
/work/include/itkLabelSetMorphBaseImageFilter.h:45:38:
note: template parameter is declared here
template< typename TInputImage, bool doDilate,
                                     ^
```

Have not found a suitable explanation/solution to the error.

Any guess?

Reply:
I guess that bool is incorrect, it should be substituted by the values true and false for the instantiation.

Reply:
Thanks @phcerdan . Specifying the template argument value certainly does the trick ! Wondering why this is necessary in this case though.